File: javascript/code-challenges/Data-Structures/linked-list/linked-list.js
class Node {
  constructor(value){
    this.value = value;
    this.next = null;
  }
}

class LinkedList {
  constructor(head = null){
    this.head = head;
    this.length = 0;
    this.tail = null
  }
  insert(data){
    const makeNode = new Node(data);
    if (this.head === null){
      this.head = makeNode;
    }
  } else {
    let current = this.head;
    while (current.next){
      current = current.next;
    }
    current.next = makeNode;
  }
};

File: python/code_challenges/linked_list/linked_list.py
class Node:
    def __init__(self, data):
        self.data = data
        self.next = None

class LinkedList:
    def __init__(self):
        self.head = self

if __name__ == '__main__':
    list = LinkedList()
    list.head = Node('Test')
    test1 = Node('test1')
    test2 = Node('test2')

